web/src/user_group_edit.ts

```diff
@@ -1450,6 +1468,10 @@ export function switch_group_tab(tab_name: string): void {
         the group_list_toggler widget.  You may instead want to
         use `group_list_toggler.goto`.
     */
+    update_displayed_groups(
+        z.string().optional().parse(filters_dropdown_widget.value()) ?? initial_group_filter,
+    );
```

Comment:
Why this needs to be called here? Left panel is anyways being redrawn after this. I might be missing something.